### PR TITLE
Client pages 04162022

### PR DESCRIPTION
**Pull request by BSaenz12**
- Implemented nav bar logic for Client (UserTypeID: 1)
- Implemented Scheduled Services and Past Services pages, which are populated with static data.

### CODE REVIEW

File: QuickFixers/Views/Client/ScheduledServices.cshtml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <td>
+            @Html.DisplayFor(modelItem => item.serviceDate)
+        </td>
+        <td>
+            @Html.DisplayFor(modelItem => item.serviceAddress)
+        </td>
+        <td>
+            @Html.ActionLink("Edit", "Edit", new { id = item.scheduledServiceID/* id=item.PrimaryKey */ }) |
+            @Html.ActionLink("Details", "Details", new { id = item.scheduledServiceID /* id=item.PrimaryKey */ }) |
+            @Html.ActionLink("Delete", "Delete", new { id = item.scheduledServiceID /* id=item.PrimaryKey */ })
```
Comment (Jody-Whitis) [inline QuickFixers/Views/Client/ScheduledServices.cshtml]: Please update links to go to an existing page like /Test or /Shared/Error(or maybe remove it we don't need any)

File: QuickFixers/Views/Client/ScheduledServices.cshtml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+@{
+    ViewBag.Title = "ScheduledServices";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h2>ScheduledServices</h2>
+
+<p>
+    @Html.ActionLink("Create New", "Create")
```
Comment (Jody-Whitis) [inline QuickFixers/Views/Client/ScheduledServices.cshtml]: Please add error handling on the link.

File: QuickFixers/Views/Client/PastServices.cshtml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+@{
+    ViewBag.Title = "PastServices";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h2>PastServices</h2>
+
+<p>
+    @Html.ActionLink("Create New", "Create")
```
Comment (Jody-Whitis) [inline QuickFixers/Views/Client/PastServices.cshtml]:
```suggestion
    @Html.ActionLink("Create New","Index","Test")
```

File: QuickFixers/Views/Client/PastServices.cshtml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        <td>
+            @Html.DisplayFor(modelItem => item.serviceDate)
+        </td>
+        <td>
+            @Html.DisplayFor(modelItem => item.serviceAddress)
+        </td>
+        <td>
+            @Html.ActionLink("Edit", "Edit", new { id=item.scheduledServiceID }) |
+            @Html.ActionLink("Details", "Details", new { id=item.scheduledServiceID }) |
+            @Html.ActionLink("Delete", "Delete", new { id=item.scheduledServiceID })
```
Comment (Jody-Whitis) [inline QuickFixers/Views/Client/PastServices.cshtml]:
```suggestion
            @Html.ActionLink("Edit", "Index","Test") |
            @Html.ActionLink("Details", "Index","Test") |
            @Html.ActionLink("Delete", "Index","Test")
```

File: QuickFixers/Views/Client/ScheduledServices.cshtml (hunk 1 above)
Comment (BSaenz12) [reply QuickFixers/Views/Client/ScheduledServices.cshtml]: Removed unnecessary links.

File: QuickFixers/Views/Client/ScheduledServices.cshtml (hunk 2 above)
Comment (BSaenz12) [reply QuickFixers/Views/Client/ScheduledServices.cshtml]: Removed unnecessary link.

File: QuickFixers/Views/Client/PastServices.cshtml (hunk 3 above)
Comment (BSaenz12) [reply QuickFixers/Views/Client/PastServices.cshtml]: Removed link.

File: QuickFixers/Views/Client/PastServices.cshtml (hunk 4 above)
Comment (BSaenz12) [reply QuickFixers/Views/Client/PastServices.cshtml]: Removed unnecessary links.

**Review by Jody-Whitis — APPROVED** [review]
Awesome Job @BSaenz12! (>' ')>

**Pull request merged**